src/state/ContextProvider.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
 
+const { remote } = window.require('electron')
+const { process } = remote
```

Comment:
Can be combined into a single line I guess.

Reply:
ACtually, remote was deprecated since Electron 12, so there is a lot of things to do to migrate to the new @electron/remote.
I'm just not sure if I want to do it in this PR

Reply:
https://github.com/electron/remote